outlook/calendar/pkg/util/util.go

```diff
@@ -18,3 +18,20 @@ func Map[T, U any](arr []T, f func(T) U) []U {
 	}
 	return out
 }
+
+func Flatten[T any](arr [][]T) []T {
+	var out []T
+	for _, v := range arr {
+		out = append(out, v...)
+	}
+	return out
+}
+
+func MapValues[T comparable, U any](m map[T]U) []U {
+	// The maps.Values function returns an iterator, but we want a slice.
```

Comment:
It doesn't seem like the implementation needs a slice, but I'm OK with not introducing iterators at this point.